src/main/java/uk/gov/hmcts/ccd/data/definition/CachedCaseDefinitionRepository.java

```diff
@@ -49,13 +49,17 @@ public UserRole getUserRoleClassifications(String userRole) {
 
     @Override
     public List<UserRole> getClassificationsForUserRoleList(List<String> userRoles) {
-        final List<String> missingRoles = userRoles.stream()
+        List<String> missingRoles = userRoles
+            .stream()
             .filter(role -> !userRoleClassifications.containsKey(role))
+            .sorted()
             .collect(Collectors.toList());
+
         if (!missingRoles.isEmpty()) {
-            final List<UserRole> missingClassifications = caseDefinitionRepository.getClassificationsForUserRoleList(missingRoles);
-            missingClassifications.forEach(userClassification -> userRoleClassifications.putIfAbsent(userClassification.getRole(), userClassification));
+            caseDefinitionRepository.getClassificationsForUserRoleList(missingRoles)
```

Comment:
I think the previous version with the explicit variable `missingClassifications` makes the code more clear. And has not impact on performance

Reply:
I found the later more readable 😄 . Anyway reverted to previous version